src/main/java/com/depromeet/archive/domain/archive/service/ArchiveService.java

```diff
@@ -0,0 +1,22 @@
+package com.depromeet.archive.domain.archive.service;
+
+import com.depromeet.archive.controller.dto.archive.ArchiveDto;
+import com.depromeet.archive.domain.archive.entity.Archive;
+import com.depromeet.archive.domain.archive.entity.ArchiveRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class ArchiveService {
+
+    private final ArchiveRepository archiveRepository;
+
+    @Transactional(readOnly = true)
+    public ArchiveDto findById(Long id) {
+        Archive archive = archiveRepository.findById(id).orElseThrow(()
+                -> new IllegalArgumentException("해당 아카이브가 없습니다."));
```

Comment:
```suggestion
        Archive archive = archiveRepository.findById(id)
            .orElseThrow(() -> new IllegalArgumentException("해당 아카이브가 없습니다."));
```
사소한건데 이런식으로 작성하는게 더 보기 좋을 것 같아요 :)